### PR TITLE
terraformer 0.7.4 (New Formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/terraformer.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/terraformer version")
+
+    help_output = "Available Commands"
+    assert_match help_output.to_s, shell_output("#{bin}/terraformer -h")
+
+    import_error = "google importing project aaa"
+    assert_match import_error.to_s, shell_output("#{bin}/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff")
```
Comment (chenrui333) [inline Formula/terraformer.rb]: @fxcoudert I am facing similar error with this line.

```
==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer version
==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer -h
==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff
2019/06/03 07:56:03 google importing project aaa
2019/06/03 07:56:03 google importing... gcs
2019/06/03 07:56:03 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
2019/06/03 07:56:03 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
Error: terraformer: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: <0> expected but was
<1>.
/Library/Ruby/Gems/2.3.0/gems/test-unit-3.1.5/lib/test/unit/assertions.rb:55:in `block in assert_block'
```

Comment (fxcoudert) [reply Formula/terraformer.rb]: It expected a zero exit code. Pass the expected exit code as second argument to `shell_output`.

Comment (chenrui333) [reply Formula/terraformer.rb]: ok, will try it now.

Comment (chenrui333) [reply Formula/terraformer.rb]: not quite working with this line
```
assert_match import_error.to_s, shell_output("#{bin}/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff", 0)

==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff
2019/06/03 09:50:14 google importing project aaa
2019/06/03 09:50:14 google importing... gcs
2019/06/03 09:50:14 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
2019/06/03 09:50:14 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
Error: terraformer: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: <0> expected but was
<1>.
/Library/Ruby/Gems/2.3.0/gems/test-unit-3.1.5/lib/test/unit/assertions.rb:55:in `block in assert_block'
```

Comment (fxcoudert) [reply Formula/terraformer.rb]: The exit code is 1, so if you expect that, pass `1` as second argument.

Comment (chenrui333) [reply Formula/terraformer.rb]: Tried that as well, not working as expected.
```
assert_match import_error.to_s, shell_output("#{bin}/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff", 1)

==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks
2019/06/04 10:00:38 google importing project aaa
2019/06/04 10:00:38 google importing... gcs
2019/06/04 10:00:38 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
2019/06/04 10:00:38 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
Error: terraformer: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: </aaa/> expected to be =~
<"">.
```

Comment (chenrui333) [reply Formula/terraformer.rb]: Adopt the similar fix from helmfile, should be good now. Will squash the changes.

**Comment by chenrui333** [comment]
```
13:24:41 ==> brew audit terraformer --online --new-formula
13:24:57 ==> FAILED
13:24:57 ==> Installing 'bundler' gem
13:24:57 Successfully installed bundler-2.0.1
13:24:57 1 gem installed
13:24:57 Error: 1 problem in 1 formula detected
13:24:57 terraformer:
13:24:57   * C: 30: col 171: Line is too long. [197/170]
13:24:57 
13:24:57 ==> brew bottle --verbose --json terraformer
13:26:29 
```

**Comment by chenrui333** [comment]
Should be good now!

**Pull request closed without merging**